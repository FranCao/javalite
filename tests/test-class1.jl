class square {
    string name;
    int side;
}

int main() {
    class square sq = square("fst_sq", 42);
    print(sq.side);
    print(sq.name);
    sq.side = 2;
    sq.name = "snd_sq";
    print(sq.side);
    print(sq.name);
    return 0;
}