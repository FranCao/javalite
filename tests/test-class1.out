42
fst_sq
2
snd_sq